syncopy/tests/test_specest.py

```diff
@@ -586,7 +599,7 @@ def test_tf_toi(self):
                 cfg.t_ftimwin = winsize
                 for toi in toiVals:
                     cfg.toi = toi
-                    tfSpec = freqanalysis(cfg, self.tfData)
+                    tfSpec = freqanalysis(cfg, TestMTMConvol.get_tfdata_mtmconvol())
```

Comment:
However, do we really need to repopulate a virgin object every time here?

Reply:
I have changed this, the data is now created before the loop and re-used within loop iterations.

Reply:
perfect, thx :+1: